doc/include/tab-repository.md

```diff
@@ -0,0 +1,128 @@
+=== "Ubuntu"
+
+    ```bash
+    apt update
+    apt -y install apt-transport-https wget gnupg
+
+    wget -O - https://packages.icinga.com/icinga.key | apt-key add -
+
+    . /etc/os-release; if [ ! -z ${UBUNTU_CODENAME+x} ]; then DIST="${UBUNTU_CODENAME}"; else DIST="$(lsb_release -c| awk '{print $2}')"; fi; \
+    echo "deb https://packages.icinga.com/ubuntu icinga-${DIST} main" > \
+    /etc/apt/sources.list.d/${DIST}-icinga.list
+    echo "deb-src https://packages.icinga.com/ubuntu icinga-${DIST} main" >> \
+    /etc/apt/sources.list.d/${DIST}-icinga.list
```

Comment:
Please consult https://packages.icinga.com/debian/ on how we shall do this instead.

Reply:
* CC https://github.com/Icinga/packages.icinga.com/pull/16